platform-sdk/swirlds-logging/src/main/java/module-info.java

```diff
@@ -28,6 +28,7 @@
     requires com.swirlds.config.extensions;
     requires com.fasterxml.jackson.core;
     requires com.fasterxml.jackson.datatype.jsr310;
+    requires com.google.common;
```

Comment:
Are we using this module at all? I can't find a reference to guava...
Either way, I am a big fan of the utility, but it would be best to avoid it for the logging framework as much as possible; it is lengthy as a dependency and full of classes marked as beta.